crates/precompile/src/bls12_381/bls12_381_const.rs

```diff
@@ -0,0 +1,68 @@
+use crate::PrecompileWithAddress;
+
+use crate::bls12_381::g1_add;
+use crate::bls12_381::g1_msm;
+use crate::bls12_381::g2_add;
+use crate::bls12_381::g2_msm;
+use crate::bls12_381::pairing;
+use crate::bls12_381::map_fp_to_g1;
+use crate::bls12_381::map_fp2_to_g2;
+
+pub const G1_ADD_ADDRESS: u64 = 0x0b;
+pub const G1_ADD_BASE_GAS_FEE: u64 = 375;
+pub const G1_ADD_INPUT_LENGTH: usize = 256;
+pub const G1_MSM_ADDRESS: u64=0x0c;
+pub const G1_MSM_BASE_GAS_FEE: u64 = 1200;
+pub const G1_MSM_INPUT_LENGTH: usize = 160;
+pub const G1_OUTPUT_LENGTH: usize = 128;
+pub const G1_INPUT_ITEM_LENGTH: usize = 128;
+pub const G2_ADD_ADDRESS: u64 = 0x0d;
+pub const G2_ADD_BASE_GAS_FEE: u64 = 600;
+pub const G2_ADD_INPUT_LENGTH: usize = 512;
+pub const G2_MSM_ADDRESS: u64 = 0x0e;
+pub const G2_MSM_BASE_GAS_FEE: u64 = 22500;
+pub const G2_MSM_INPUT_LENGTH: usize = 288;
+pub const G2_OUTPUT_LENGTH: usize = 256;
+pub const G2_INPUT_ITEM_LENGTH: usize = 256;
+pub const PAIRING_ADDRESS: u64 = 0x0f;
+pub const PAIRING_PAIRING_MULTIPLIER_BAS: u64 = 32600;
+pub const PAIRING_PAIRING_OFFSET_BASE: u64 = 37700;
+pub const PAIRING_INPUT_LENGTH: usize = 384;
+pub const MAP_FP_TO_G1_ADDRESS: u64 = 0x10;
+pub const MAP_FP_TO_G1_BASE_GAS_FEE: u64 = 5500;
+pub const MAP_FP2_TO_G2_ADDRESS: u64 = 0x11;
+pub const MAP_FP2_TO_G2_BASE_GAS_FEE: u64 = 0x23800;
+pub const MSM_MULTIPLIER: u64 = 1000;
+/// Number of bits used in the BLS12-381 curve finite field elements.
+pub const UTILS_NBITS: usize = 256;

```

Comment:
You dont needs to call them `UTILS_NBITS`, `NBITS` is fine

same with all other consts